images/image-repo-list-ws2019

```diff
@@ -0,0 +1,8 @@
+dockerLibraryRegistry: claudiubelu
```

Comment:
Is this just a temporary solution, or do we expect to always need different repo-list files for different Windows versions?

I see 'e2eteam' in a number of places in the scripts under https://github.com/kubernetes-sigs/windows-testing/tree/master/images; is the mismatch with this file problematic?

Reply:
I'm not sure how these will be managed long-term. @PatrickLang has more idea on that.

Moving the files to this repository is a good first step anyway.

Reply:
Sooo, basically, at the moment, you cannot run 1803 container images on 2019 and vice-versa, hence the 2 different repositories: e2eteam having 1803 images and claudiubelu having WS 2019 images. Ideally, we wouldn't have to rely on different files / repos, but LGTM for the time being. We could have a TODO to see how we can have both types of images under the same repo.

As far 'e2eteam' being in the images, that's fine, that tag is the "default" value when building / tagging new images, but since it is an argument to BuildImages.ps1, a different value can be passed instead. No reason to change it ATM.

Reply:
I think different filenames is ok for now.

From my understanding, Docker EE 18.09 can choose the right container image if we push a multiarch manifest with different versions for platform."os.version".

run `docker manifest inspect microsoft/dotnet:2.2-runtime` for an example of an image supporting both 1803 and 1809/2019

```
{
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1984,
         "digest": "sha256:c63344952bb7ad52e916ca31fb9292f6793a11dc41e91bb752fb9b2fd953f07d",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.17134.523"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 1985,
         "digest": "sha256:4d9eadcee45782a2dabc15b940a7ac336efb0524117c3a440835679b554b6031",
         "platform": {
            "architecture": "amd64",
            "os": "windows",
            "os.version": "10.0.17763.253"
         }
      },
```

Reply:
I added the multiarch requirement to #3